app/controllers/whitelist_controller.rb

```diff
@@ -0,0 +1,77 @@
+class WhitelistController < ApplicationController
```

Comment:
It's a rails convention that controller name should be the plural format of model name, and since rails is a 'convention over configuration' framework most people will assume controller name is "AuthorizedUsersController" if model name is "AuthorizedUser"